src/main/resources/org/jenkinsci/plugins/github/config/GitHubPluginConfig/config.groovy

```diff
@@ -26,8 +26,7 @@ f.section(title: descriptor.displayName) {
                 table(width: "100%", style: "margin-left: 7px;") {
                     f.optionalBlock(title: _("Specify another hook URL for GitHub configuration"),
                             inline: true,
-                            field: "overrideHookUrl",
-                            checked: instance.overrideHookURL) {
+                            checked: instance.isOverrideHookUrl) {
```

Comment:
either method call either method call via field name, i think it wouldn't work

Reply:
JCasC does say no to optional blocks, use optional property https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md#step-4

Reply:
Or move it out of the optional block, it smells :cry:

Reply:
This isn't part of my wheel house, I think the way I have it setup, conforms to the example here: https://wiki.jenkins.io/display/JENKINS/Jelly+form+controls

I'm removing the "field" from the toggling of the checkbox to just toggle it based upon the value,  I'll see if I can add a test to verify that the checkbox is "checked" based upon the current code or correct if that doesn't work.

Reply:
`isOverrideHookUrl()` or `instance.overrideHookURL`  code had correct call